add README files 2/8 (#5766)
TheAlgorithms/Python: conversions/README.md

# Conversion

> The act to convert a type of data, a number from a numerical base or unit into one of another type, base, unit.

Reviewer: ```suggestion
Conversion programs convert a type of data, a number from a numerical base or unit into one of another type, base, unit.
```
